python-sdk/nuscenes/eval/prediction/README.md

```diff
@@ -1,5 +1,6 @@
 # nuScenes prediction task
 In this document we present the rules, submission process, and evaluation metrics of the nuScenes prediction task.
```

Comment:
I'd remove this text. It' not necessary IMO.

Reply:
Done